src/nvim/ui_compositor.c

```diff
@@ -535,6 +544,11 @@ static void ui_comp_raw_line(UI *ui, Integer grid, Integer row,
   } else {
     compose_debug(row, row+1, startcol, endcol, dbghl_normal, false);
     compose_debug(row, row+1, endcol, clearcol, dbghl_clear, true);
+#ifndef NDEBUG
+    for (int i = 0; i < endcol-startcol; i++) {
```

Comment:
@bfredl I wonder if https://github.com/neovim/neovim/issues/14637 is due to `i` possibly being out of bounds in `attrs`. Shouldn't `i` start at some offset instead of assuming `0`? I see that in https://github.com/neovim/neovim/blob/fce0d54eb2bcfff6903649ab093603dcd8f21f23/src/nvim/ui.c#L450 `attrs` is offset by `off`.